src/content/ko/2020/README.md

```diff
@@ -0,0 +1 @@
+This is the directory in which content teams add the markdown version of their chapters.
```

Comment:
Hmmm... this file shouldn't exist. But see it does exist for English directory. It shouldn't so will remove. Can you remove this ko file?